include/Output.hpp

```diff
@@ -1,13 +1,25 @@
 #pragma once
 
 # include "Wlroots.hpp"
+# include "Listeners.hpp"
 
 class Server;
 
-struct Output
+
+class Output : public Listeners::OutputListeners
 {
+public:
+  Output(Server *server, struct wlr_output *wlr_output);
+  ~Output() = default;
+
   struct wl_list link;
+
+  void setFrameListener();
+
+private:
   Server *server;
   struct wlr_output *wlr_output;
-  struct wl_listener frame;
+
+private:
```

Comment:
Why is private present twice?

Reply:
Just a simple mistake that went throuhgh. See issue #9 that had been made to manage this kind of thing, among other things.

Reply:
ok for me.